cryptography

```diff
@@ -0,0 +1,15 @@
+---
+Title: 'Cryptography' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'A method of protecting information and communications through the use of codes that allow only the sender and receiver of a message to view its contents.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
```

Comment:
```suggestion
Description: 'Cryptography is a method of protecting information and communications through codes that allow only the sender and receiver of a message to view its contents.'
```

Reply:
Rule of thumb for an entry `Description`:

- For concept entries (which this is), it starts with the noun/concept.
- For term entries (that fall under concepts), it starts with a present-tense verb (e.g., "Returns ...")